pkg/controllers/repositoryserver/repositoryserver_controller.go

```diff
@@ -87,21 +87,53 @@ func (r *RepositoryServerReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	if err = r.Status().Update(ctx, repoServerHandler.RepositoryServer); err != nil {
 		return ctrl.Result{}, err
 	}
+	logger.Info("Setup RepositoryServer, Create or update owned resources")
```

Comment:
Same here. Either move it to debug or remove the logs. 
Also, how do you differentiate logs of different CRs?